### ISSUE TITLE
Support Crystal 1.0.0

### ISSUE THREAD

**Issue by plambert** [hutou/tablo#1]
What is needed to support Crystal 1.0.0?  Currently, the `crystal:` declaration in `shard.yml` is preventing `shard install` or `shard update` without the `--ignore-crystal-version` flag.

**Comment by plambert** [comment]
```
$ shards install
Resolving dependencies
Fetching https://github.com/hutou/tablo.git
Unable to satisfy the following requirements:

- `crystal (~> 0.36, >= 0.36.1)` required by `tablo 0.9.5`
Failed to resolve dependencies, try updating incompatible shards or use --ignore-crystal-version as a workaround if no update is available.
```

**Comment by hutou** [comment]
Just published release 0.9.6 for Crystal 0.36.1 and  above
